contracts/Registry.vy

```diff
@@ -1,4 +1,4 @@
-# @version 0.2.11
+# @version 0.2.12
```

Comment:
We can't bump this without redeploying the Registry. Unless there is a bug or issue we are vulnerable to with this contract, best to leave this off the table.